src/guides/v2.2/cloud/cdn/fastly-vcl-blacklist.md

```diff
@@ -50,6 +50,28 @@ Create an `blocklist.json` file with the following JSON content:
 }
 ```
 
+## Sample Block By Country Code (A two-character ISO 3166-1 country code for the country associated with the IP address.)
+```json
+{
+  "name": "blockbycountrycode",
+  "dynamic": "0",
+  "type": "recv",
+  "priority": "5",
+  "content": "if ( geoip.country_code == \"HK\" ) { error 405 \"Not allowed\";}"
+}
+```
+
+## Sample Block By http.User-Agent
```

Comment:
```suggestion
**Sample Block By http.User-Agent:**

```

Reply:
Will make these suggested changes in a separate PR.